ui/pom.xml

```diff
@@ -28,20 +28,20 @@
   <artifactId>integration-azure-oauth-ui</artifactId>
   <packaging>xar</packaging>
 
-  <name>Microsoft Azure Active Directory Single-Sign-On (SSO) (pro)</name>
-  <description>This extension allows users to authenticate to the wiki using Azure Active Directory. The app can be purchased individually or part of the XWiki Pro package. Try it free.</description>
+  <name>Microsoft Entra ID OpenID Connect (OIDC)</name>
```

Comment:
I think this is the name displayed in EM. Probably better to check with @snazare about what name should we use